docs/api/orders/submit-orders.api.mdx

```diff
@@ -5,7 +5,7 @@ description: "Submits an order to Bonsai for processing. Be careful as these ord
 sidebar_label: "Submit Orders"
```

Comment:
Not sure why this file changed? Did we forget to add anything last time? It's really annoying to see the git diff if these are unformatted